src/traces/scattergl/index.js

```diff
@@ -863,9 +863,11 @@ function plot(container, subplot, cdata) {
 
                 fillOptions.opacity = trace.opacity;
                 fillOptions.positions = pos;
+
+                return fillOptions;
             });
 
-            scene.fill2d.update(scene.fillOptions);
+            scene.fill2d.update(fillOptions);
```

Comment:
You forgot to `git add` the json mock corresponding to `gl2d_line_select.png`

Reply:
This is the one that accidentally made it into master, as mentioned above
> By accident pushed 41c067a to master, this should fix that.

But since I had that in master as the base of my own PR, I removed it there https://github.com/plotly/plotly.js/pull/2379#discussion_r168913028